File: 2003/VB101SamplesAll/VB101SamplesWebDevelopment/DataAccess/App_Code/Product.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Data
Imports System.Configuration
Imports System.Web
Imports System.Web.Security
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts
Imports System.Web.UI.HtmlControls

Namespace Microsoft.Samples.Web.DataAccess

    Public Class Product

        Public Sub New()
        End Sub
        Private _productId As Integer

        Public Property ProductId() As Integer
            Get
                Return _productId
            End Get
            Set(ByVal value As Integer)
                _productId = value
            End Set
        End Property
        Private _name As String

        Public Property Name() As String
            Get
                Return _name
            End Get
            Set(ByVal value As String)
                _name = value
            End Set
        End Property
        Private _productNumber As String

        Public Property ProductNumber() As String
            Get
                Return _productNumber
            End Get
            Set(ByVal value As String)
                _productNumber = value
            End Set
        End Property
        Private _reorderPoint As Integer

        Public Property ReorderPoint() As Integer
            Get
                Return _reorderPoint
            End Get
            Set(ByVal value As Integer)
                _reorderPoint = value
            End Set
        End Property
        Private _standardCost As Decimal

        Public Property StandardCost() As Decimal
            Get
                Return _standardCost
            End Get
            Set(ByVal value As Decimal)
                _standardCost = value
            End Set
        End Property
        Private _listPrice As Decimal

        Public Property ListPrice() As Decimal
            Get
                Return _listPrice
            End Get
            Set(ByVal value As Decimal)
                _listPrice = value
            End Set
        End Property
    End Class
End Namespace

File: 2003/CS101SamplesAll/CS101SamplesWebDevelopment/DataAccess/App_Code/Product.cs
using System;
using System.Data;
using System.Configuration;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;

namespace Microsoft.Samples.Web.DataAccess
{
	/// <summary>
	/// Summary description for Product
	/// </summary>
	public class Product
	{
		public Product()
		{
		}

		private int _productId;
		public int ProductId
		{
			get { return _productId; }
			set { _productId = value; }
		}

		private string _name;
		public string Name
		{
			get { return _name; }
			set { _name = value; }
		}

		private string _productNumber;
		public string ProductNumber
		{
			get { return _productNumber; }
			set { _productNumber = value; }
		}

		private int _reorderPoint;
		public int ReorderPoint
		{
			get { return _reorderPoint; }
			set { _reorderPoint = value; }
		}

		private decimal _standardCost;
		public decimal StandardCost
		{
			get { return _standardCost; }
			set { _standardCost = value; }
		}

		private decimal _listPrice;
		public decimal ListPrice
		{
			get { return _listPrice; }
			set { _listPrice = value; }
		}
	}
}
